.github/workflows/create-demo-clusters.yml

```diff
@@ -279,28 +302,54 @@ jobs:
           PAGERDUTY_INTEGRATION_KEY: ${{ secrets.RELEASE_MANAGEMENT_PAGERDUTY_INTEGRATION_KEY }}
           REGISTRY_USERNAME: ${{ secrets.QUAY_RHACS_ENG_RO_USERNAME }}
           REGISTRY_PASSWORD: ${{ secrets.QUAY_RHACS_ENG_RO_PASSWORD }}
+          STACKROX_DIR: ${{ github.workspace }}
+          #local_script_url: /repos/${{github.repository}}/contents/.github/workflows/scripts/start-acs.sh?ref=${{ github.ref_name }}
+          local_script_url: /repos/stackrox/actions/contents/.github/workflows/scripts/start-acs.sh?ref=jv-ROX-17741-long-running-cluster-should-include-collector
         run: |
-          set -uo pipefail
-          ./deploy/k8s/central.sh
-          kubectl -n stackrox port-forward deploy/central 8000:8443 > /dev/null 2>&1 &
-          sleep 20
-
-          ./deploy/k8s/sensor.sh
+          gh api -H "$ACCEPT_RAW" "$local_script_url" | bash -s
```

Comment:
Why don't we call the script directly? There is no need to use `gh api` anymore to download the scripts as they are now part of the action.

Reply:
This now calls another action which calls the script.